apps/devhub/tasks.py

```diff
@@ -165,12 +165,13 @@ def validate_file(file_id, hash_, **kw):
 def handle_upload_validation_result(results, upload_pk, annotate=True):
     """Annotates a set of validation results, unless `annotate` is false, and
     saves them to the given FileUpload instance."""
-    if annotate:
-        results = annotate_validation_results(results)
+    with statsd.timer('devhub.validator.handle_upload_validation_result'):
```

Comment:
We already have timing for each task. You can see this one at https://app.datadoghq.com/metric/explorer?live=true&page=0&is_auto=false&from_ts=1452625319843&to_ts=1452628919843&tile_size=m&exp_metric=addons_prod.tasks.devhub.tasks.handle_upload_validation_result.avg&exp_scope=&exp_agg=avg&exp_row_type=metric